src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String CSV_SEPARATOR = ",";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private int buy;
+    private int supply;
+
+    public String getResult() {
```

Comment:
should all methods be public?
https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents